### PR TITLE
Updated to latest presto version

### PR DESCRIPTION
**Pull request by ChethanUK**
Presto SQL version update..

Latest Repo of Presto is https://repo1.maven.org/maven2/io/prestosql/presto-server/307/presto-server-307.tar.gz

Trying to solve: https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/issues/492

### CODE REVIEW

**Comment by ChethanUK** [comment]
I signed it!

File: presto/presto.sh [hunk 1]
```diff
@@ -228,7 +229,8 @@ function configure_and_start_presto(){
   if [[ "${HOSTNAME}" == "${PRESTO_MASTER_FQDN}" ]]; then
     configure_master
     start_presto
-    wait_for_presto_cluster_ready
+    # Need to fix following func: Because system.runtime.nodes is giving wrong address
```
Comment (functicons) [inline presto/presto.sh]: We must wait for Presto cluster to be ready, please fix it.

File: presto/presto.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ function wait_for_presto_cluster_ready() {
 
 function get_presto(){
   # Download and unpack Presto server
-  wget https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
+  wget https://repo1.maven.org/maven2/io/prestosql/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
```
Comment (functicons) [inline presto/presto.sh]: Can you provide more information about io.prestosql? e.g., project homepage. How is it different from com.facebook.presto?

File: presto/README.MD [hunk 3]
```diff
@@ -6,11 +6,15 @@ This initialization action installs the latest version of [Presto](prestodb.io)
 You can use this initialization action to create a new Dataproc cluster with Presto installed:
 
 1. Using the `gcloud` command to create a new cluster with this initialization action. The following command will create a new cluster named `<CLUSTER_NAME>`.
-
     ```bash
     gcloud dataproc clusters create <CLUSTER_NAME> \
       --initialization-actions gs://dataproc-initialization-actions/presto/presto.sh
     ```
+    Or
```
Comment (functicons) [inline presto/README.MD]: By convention, this is not needed.

File: presto/presto.sh (hunk 2 above)
Comment (findepi) [reply presto/presto.sh]: @functicons, please see @martint's (one of the creators of presto) explanation: https://github.com/prestosql/presto/issues/380#issuecomment-469525303

**Comment by medb** [comment]
We already have [Starbust Presto init action](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/starburst-presto) for Presto SQL

**Pull request closed without merging**